README.md

```diff
@@ -62,6 +63,8 @@ Video](https://img.youtube.com/vi/-Xd84hlIjkI/0.jpg)](https://www.youtube.com/wa
 
 - `FLASK_STATIC_DIGEST_HOST_URL` has been added to configure an optional external host, aka. CDN ([explained here](#configuring-this-extension))
 - If your blueprints have static files they will get digested now too (including nested blueprints!)
+- Optional Brotli support has been added, `pip install Flask-Static-Digest[brotli]`
```

Comment:
Can you remove this line?

Reply:
I would have thought as you have a list of things that changed after your video, this would count as a change?

Reply:
Ok, we can keep it, but we can we change it to:

- Optional Brotli support has been added

The installation steps are listed in the installation section.

Reply:
This change is still pending.